### PR TITLE
Add missingIf option to other functions

### PR DESCRIPTION
**Pull request by yoannmoinet**
Following #33.
Other predicates have the same issue.
Follow the same logic for them too, adding tests and doc as well.

### CODE REVIEW

File: sources/predicates/helperPredicates.ts [hunk 1]
```diff
@@ -123,20 +123,30 @@ export function isNullable<T extends AnyStrictValidator>(spec: T) {
...
+  falsy: (keys, key, value) => keys.has(key) && !!value[key],
+};
+
 /**
  * Create a validator that checks that the tested object contains the specified
  * keys.
- */
-export function hasRequiredKeys(requiredKeys: string[]) {
+*/
+export function hasRequiredKeys(requiredKeys: string[], options?: { missingIf: MissingType }) {
```
Comment (arcanis) [inline sources/predicates/helperPredicates.ts:139]:
```suggestion
export function hasRequiredKeys(requiredKeys: string[], options?: { missingIf?: MissingType }) {
```

File: sources/predicates/helperPredicates.ts [hunk 2]
```diff
@@ -151,14 +161,15 @@ export function hasRequiredKeys(requiredKeys: string[]) {
 * Create a validator that checks that the tested object contains at least one
 * of the specified keys.
 */
-export function hasAtLeastOneKey(requiredKeys: string[]) {
+export function hasAtLeastOneKey(requiredKeys: string[], options?: { missingIf: MissingType }) {
```
Comment (arcanis) [inline sources/predicates/helperPredicates.ts:164]:
```suggestion
export function hasAtLeastOneKey(requiredKeys: string[], options?: { missingIf?: MissingType }) {
```

File: sources/predicates/helperPredicates.ts [hunk 3]
```diff
@@ -170,17 +181,18 @@ export function hasAtLeastOneKey(requiredKeys: string[]) {
 /**
  * Create a validator that checks that the tested object contains none of the
  * specified keys.
- */
-export function hasForbiddenKeys(forbiddenKeys: string[]) {
+*/
+export function hasForbiddenKeys(forbiddenKeys: string[], options?: { missingIf: MissingType }) {
```
Comment (arcanis) [inline sources/predicates/helperPredicates.ts:185]:
```suggestion
export function hasForbiddenKeys(forbiddenKeys: string[], options?: { missingIf?: MissingType }) {
```

**Comment by liuxingbaoyu** [comment]
Hi, can you release a new version? Thanks!

**Comment by arcanis** [comment]
It's already released 🙂

**Pull request merged**